src/app/services/role-guard.service.ts

```diff
@@ -1,35 +0,0 @@
-import { Injectable } from '@angular/core';
-import { ActivatedRouteSnapshot, Router } from '@angular/router';
-import { AuthService } from './auth.service';
-
-@Injectable({
```

Comment:
This is for protecting routes. But I guess it makes sense that you removed these since it would be useless if you removed the other routes too.